code/game/gamemodes/gang/gang_things.dm

```diff
@@ -18,7 +18,8 @@
 	if(user.mind.has_antag_datum(/datum/antagonist/ert/families))
 		to_chat(user, "As a police officer, you can't join this family. However, you pretend to accept it to keep your cover up.")
 		for(var/threads in team_to_use.free_clothes)
-			new threads(get_turf(user))
+			var/obj/O = new threads(get_turf(user))
+			O.armor = list("melee" = 20, "bullet" = 20, "laser" = 20, "energy" = 20, "bomb" = 20, "bio" = 20, "rad" = 20, "fire" = 20, "acid" = 20, "wound" = 20)
```

Comment:
that wound armor of 20 is a bit high, especially since you can stack a gang jumpsuit and a gang suit (or even just normal armor) for double the armor (or a gang mask and a gang hat for your head armor). our combat is mostly stun-based anyway, but necromanceranne might throw a fit when they see this.

also, this affects the balance between the various gangs, since some gangs receive more gang clothing (or gang clothing that covers better regions/occupies better slots) than other gangs.

Reply:
I want to see how this plays out first before tweaking it.

Reply:
Can you keep armor in suits instead of instead of jumpsuits to avoid stacking?

Reply:
Not all gangs have suit slot items and the point is to encourage wearing all of the gang outfit, not just the suit.

Reply:
give it different benefits other than armor then
jumpsuit armor causes armor stacking and it really messes up balance

Reply:
There's literally no other benefits I can give it. I made wearing your outfit give points to the gang. That wasn't enough. I'm dropping it to 7.5% armor instead on each piece, meaning you'll reach a maximum of 15% armor if wearing your full gang kit.

Reply:
this'll be resolved when armor stacking is removed entirely